### DISCUSSION TITLE
Can we do sliding window inference if my model have two inputs?

### DISCUSSION THREAD

**Discussion by luciliang** [Project-MONAI/tutorials#1029] [Q&A] [answered]
My model have two input tensors, how to do sliding window inference?

**Comment by KumoLiu** [comment, marked as answer]
Hi @luciliang, so far I find an ugly way to do this, if your two input images are the same size, then you can concat them together before sending them to the model and separate them at the beginning of the network. 
Hope it can help you, thanks!

**Reply by luciliang** [reply]
Thanks, it helps a lot!
